Sources/AppcuesKit/Presentation/UI/ExperienceData.swift

```diff
@@ -37,6 +37,31 @@ internal class ExperienceData {
 
 @available(iOS 13.0, *)
 extension ExperienceData {
+
+    enum Validator: Equatable {
+        /// The item has some value.
+        case nonEmpty
```

Comment:
This could be replaced by the `.minSelections(1)`, but I like the clarity of `.nonEmpty` and it makes more sense for a textInput where "selections" isn't a thing.